examples/example-gateway/build/endpoints/contacts/workflow/contacts_contacts_method_savecontacts.go

```diff
@@ -0,0 +1,41 @@
+// Code generated by zanzibar
+// @generated
+
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package workflow
+
+import (
+	"context"
+
+	zanzibar "github.com/uber/zanzibar/runtime"
+
+	endpointsContactsContacts "github.com/uber/zanzibar/examples/example-gateway/build/gen-code/endpoints/contacts/contacts"
+)
+
+// ContactsSaveContactsWorkflow defines the interface for ContactsSaveContacts workflow
+type ContactsSaveContactsWorkflow interface {
+	Handle(
+		ctx context.Context,
+		reqHeaders zanzibar.Header,
+		r *endpointsContactsContacts.SaveContactsRequest,
+	) (*endpointsContactsContacts.SaveContactsResponse, zanzibar.Header, error)
```

Comment:
Yes, this will be the interface between boilerplate and business code. Custom workflow will need provide a constructor `NewContactsSaveContatcsWorkflow` that returns above interface to wire things up. Generated workflow works the same way, it decouples the handler and the workflow implementation detail.